### DISCUSSION TITLE
Nuxtjs supports vue 3?

### DISCUSSION THREAD

**Discussion by JenuelDev** [nuxt/nuxt#8714] [Questions (Nuxt 2)] [answered]
Does NuxtJs supports vue 3?

**Comment by TheAlexLichter** [comment, marked as answer]
Not yet. Follow https://github.com/nuxt/nuxt.js/issues/5708 for more info :)

**Reply by JenuelDev** [reply]
wow, #5708 is a question from 2019, it's 2021 now.
